src/services/auth/plugins/mobile/schemas.ts

```diff
@@ -52,24 +83,46 @@ export const mPasswordLogin = {
     },
     { additionalProperties: false },
   ),
+  response: {
+    [StatusCodes.OK]: Type.Object(
+      { resource: Type.String({ format: 'uri' }) },
+      { description: 'Successful Response' },
+    ),
+    '4xx': errorSchemaRef,
+  },
 } as const satisfies FastifySchema;
 
 export const mauth = {
+  tags: ['authentication', 'mobile'],
+  summary: 'Authentication validating the token',
+  description: 'Authenticate to obtain session cookie given provided token and verifier',
   body: Type.Object(
     {
       [SHORT_TOKEN_PARAM]: Type.String({ format: 'jwt' }),
       verifier: Type.String(),
     },
     { additionalProperties: false },
   ),
+  response: {
+    [StatusCodes.OK]: authTokensPairSchemaRef,
+    '4xx': errorSchemaRef,
+  },
 } as const satisfies FastifySchema;
 
 export const authWeb = {
+  tags: ['authentication', 'mobile'],
+  summary: 'Authentication on the web with mobile token',
+  description:
+    'Obtain session cookie to authenticate on the web given provided mobile JWT token. Redirect to given url.',
   querystring: Type.Object(
     {
       [TOKEN_PARAM]: Type.String({ format: 'jwt' }),
```

Comment:
Also, we can add a small comment to each params to describe what it is. [Sources...](https://quobix.com/vacuum/rules/descriptions/oas3-parameter-description/)

I don't think we should put a lot of effort on this one, but can still be relevant.
I'm also wondering, what about querystrings/body.. We should probably also add comments, but didn't found any rules about it.